### PR TITLE
Account deletion at NextDNS, Nubank, Neon Pagamentos and FreeBitco.in

### PR DESCRIPTION
**Pull request by SnwMds**
<h3 align='center'>Changes</h3>

<h4 align='center'>Added</h4>

* **NextDNS** account deletion info
* **Nubank** account deletion info
* **Neon Pagamentos** account deletion info
* **FreeBitco.in** account deletion info

<h4 align='center'>Updated</h4>

* **Car2Go** account deletion info
* **Freenom** account deletion info
* **Muambator** account deletion info
* **textPlus** account deletion info
* **Mozilla Developer Network (MDN)** account deletion info
* **Amazon** account deletion info
* **Avira** account deletion info
* **Basecamp** account deletion info
* **Bodybuilding** account deletion info
* **Pluralsight (previously Code School)** account deletion info
* **CrashPlan** account deletion info
* **Cybrary** account deletion info
* **Digg** account deletion info
* **Droplr** account deletion info
* **Dwolla** account deletion info
* **eBonus** account deletion info
* **eDreams** account deletion info
* **Flattr** account deletion info
* **Freelancer** account deletion info
* **Gauges** account deletion info
* **Gamespot** account deletion info
* **MixCloud** account deletion info
* **MyJDownloader** account deletion info
* **Personello Germany** account deletion info
* **Wix** account deletion info

<h4 align='center'>Removed</h4>
 
* **BoerseBZ** account deletion info 
    - Based on information obtained from [Wikipedia](http://de.wikipedia.org/wiki/Boerse.bz), the BoerseBZ website closed down in 2014.
* **Hypejar** account deletion info 
    - The domain (`hypejar.com`) appears to be for sale. Probably not renewed. [Hypejar's official Twitter account](http://nitter.net/Hypejar) has not been active since 2015.
* **Koding** account deletion info 
    - When attempting to login or create an account, the following warning is displayed: _"Hosted version of Koding is no longer available."_

<h3 align='center'>Additional information</h3>

While accessing the other account entries in `justdeleteme.xyz`, I encountered the following "issues":

<h4 align='center'>Certificate expired</h4>

Some domains have their certificate expired (some have expired for more than 1 year), which prevented me from accessing content and verifying that account deletion information is still valid.

<h4 align='center'>Information that is incorrect or no longer exists</h4>

Some contact information/URLs provided for some account entries were invalid or no longer accessible. I tried to look for valid contact information and also tried to look for account deletion information for these account entries, both on the website itself, as well as on search engines. I was able to update most of the information that was out of date, however I could not find valid information regarding some specific account entries.

<h4 align='center'>Language</h4>

Some websites were in languages ​​I didn't understand, so it was impossible for me to verify that the contact information and/or account deletion information for some entries was still valid.

<h4 align='center'>Restrictions</h4>

Some websites simply blocked my access attempt (probably geolocation/IP restriction), which prevented me from checking if some account deletion information was still valid.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -243,7 +243,7 @@
 
     {
         "name": "Amazon",
-        "url": "https://www.amazon.com/gp/help/customer/contact-us/ref=cu_cf_email?ie=UTF8&mode=email#a",
+        "url": "http://amazon.com/gp/help/customer/contact-us",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://amazon.com/gp/help/customer/contact-us",
```

File: _data/sites.json [hunk 2]
```diff
@@ -1998,8 +1992,10 @@
 
     {
         "name": "Droplr",
-        "url": "https://d.pr/account",
+        "url": "http://d.pr/settings/profile",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://d.pr/settings/profile",
```

File: _data/sites.json [hunk 3]
```diff
@@ -2598,7 +2604,7 @@
 
     {
         "name": "Freelancer",
-        "url": "http://www.freelancer.com/report/contact.php",
+        "url": "http://freelancer.com/support/employer/general",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://freelancer.com/support/employer/general",
```

File: _data/sites.json [hunk 4]
```diff
@@ -2716,15 +2722,16 @@
 
     {
         "name": "Gauges",
-        "url": "http://get.gaug.es/tos/",
+        "url": "http://gaug.es",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://gaug.es",
```

File: _data/sites.json [hunk 5]
```diff
@@ -2689,7 +2695,7 @@
 
     {
         "name": "Gamespot",
-        "url": "https://www.gamespot.com/help/#toc-0-14",
+        "url": "http://gamespot.com",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://gamespot.com",
```

File: _data/sites.json [hunk 6]
```diff
@@ -4740,8 +4729,10 @@
 
     {
         "name": "MixCloud",
-        "url": "http://www.mixcloud.com/myaccount/account/",
+        "url": "http://mixcloud.com/settings/delete-account",
```
Comment (Nightfirecat) [inline _data/sites.json]: It looks like this is the URL which is redirected to?
```suggestion
        "url": "https://www.mixcloud.com/settings/delete-account/",
```

File: _data/sites.json [hunk 7]
```diff
@@ -4740,8 +4729,10 @@
...
         "difficulty": "easy",
+        "notes": "Click at \"Just delete my account\".",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "notes": "Click on \"Just delete my account\".",
```

File: _data/sites.json [hunk 8]
```diff
@@ -5595,10 +5619,13 @@
 
     {
         "name": "Personello Germany",
-        "url": "http://de.personello.com/mypersonello/index/feedback",
+        "url": "http://de.personello.com/service",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://de.personello.com/service",
```

File: _data/sites.json [hunk 9]
```diff
@@ -7878,8 +7920,8 @@
 
     {
         "name": "Wix",
-        "url": "http://www.wix.com/support/html5/billing/premium-plans/faq/cancel-account-20",
-        "difficulty": "easy",
+        "url": "http://support.wix.com/en/article/gdpr-permanently-deleting-your-wix-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://support.wix.com/en/article/gdpr-permanently-deleting-your-wix-account",
```

File: _data/sites.json [hunk 10]
```diff
@@ -7601,9 +7643,9 @@
 
     {
         "name": "Vidyard",
-        "url": "https://knowledge.vidyard.com",
-        "difficulty": "medium",
-        "notes": "Click \"Get Help\" and request an account deletion, entering your email into the request form.",
+        "url": "http://knowledge.vidyard.com/hc/en-us/requests/new",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://knowledge.vidyard.com/hc/en-us/requests/new",
```

File: _data/sites.json [hunk 11]
```diff
@@ -7012,9 +7052,11 @@
 
     {
         "name": "Tango",
-        "url": "http://support.tango.me/entries/23534011-How-do-I-delete-my-Tango-account-",
-        "difficulty": "hard",
-        "notes": "In order to delete your account, you have to fill out the form.",
+        "url": "http://help.tango.me/en/articles/2985296-how-do-i-delete-my-tango-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://help.tango.me/en/articles/2985296-how-do-i-delete-my-tango-account",
```

File: _data/sites.json [hunk 12]
```diff
@@ -5041,6 +5032,17 @@
         ]
     },
 
+    {
+        "name": "Neon Pagamentos",
+        "url": "http://suporte.neon.com.br/hc/pt-br/articles/360019117253-Quero-cancelar-minha-conta-Como-fa%C3%A7o",
```
Comment (Nightfirecat) [inline _data/sites.json]: Can the commit adding this entry be squashed with the commit adjusting the URL encoding? I'd prefer those changes both be incorporated in a single commit.

File: _data/sites.json [hunk 13]
```diff
@@ -527,9 +543,10 @@
 
     {
         "name": "Avira",
-        "url": "https://answers.avira.com/en/question/how-do-i-delete-my-avira-account-12214?sh=true",
-        "difficulty": "hard",
-        "note": "Contact support to request deletion of your account with the email: support@avira.com",
+        "url": "http://support.avira.com/hc/en-us/articles/360000753557--How-can-I-delete-my-Avira-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://support.avira.com/hc/en-us/articles/360000753557--How-can-I-delete-my-Avira-account",
```

File: _data/sites.json [hunk 14]
```diff
@@ -614,13 +631,15 @@
 
     {
         "name": "Basecamp",
-        "url": "https://basecamp.com/help/guides/account/cancel",
+        "url": "http://3.basecamp-help.com/article/156-cancel-your-basecamp-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://3.basecamp-help.com/article/156-cancel-your-basecamp-account",
```

File: _data/sites.json [hunk 15]
```diff
@@ -817,31 +836,20 @@
 
     {
         "name": "Bodybuilding",
-        "url": "http://www.bodybuilding.com/store/help/delete-bodyspace-account.htm",
+        "url": "http://bodybuilding.com/help?deactivate-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://www.bodybuilding.com/help?account-inquiries&deactivate-account",
```

File: _data/sites.json [hunk 16]
```diff
@@ -1045,12 +1053,14 @@
     },
 
     {
-        "name": "car2go",
-        "url": "https://www.car2go.com/US/en/#footer-contactus",
+        "name": "Car2Go",
+        "url": "http://car2go.com/US/en/contact",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://www.car2go.com/US/en/contact/",
```

File: _data/sites.json [hunk 17]
```diff
@@ -1497,12 +1491,14 @@
 
     {
         "name": "CrashPlan",
-        "url": "http://support.crashplan.com/",
+        "url": "http://helpdesk.code42.com",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://helpdesk.code42.com/hc/en-us",
```

File: _data/sites.json [hunk 18]
```diff
@@ -1599,9 +1595,10 @@
 
     {
         "name": "Cybrary",
-        "url": "https://www.cybrary.it/members",
+        "url": "http://app.cybrary.it/settings/account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://app.cybrary.it/settings/account",
```

File: _data/sites.json [hunk 19]
```diff
@@ -1807,18 +1804,13 @@
...
-        "difficulty": "hard",
-        "notes": "Contact Digg’s customer support and request for your account to be closed.",
-        "notes_fr": "Contactez le support Digg et demandez la suppresion du compte.",
-        "notes_it": "Contatta il servizio clienti e richiedi che il tuo account sia chiuso.",
-        "notes_de": "Kontaktiere Digg's Kundensupport und beantrage die Löschung deines Accounts.",
-        "notes_pt_br": "Contate a assistência ao cliente da Digg e peça que sua conta seja fechada.",
-        "notes_cat": "Contacti amb l'assistència al client de Digg i demani que es clausuri el seu compte.",
-        "notes_es": "Contacta con asistencia al cliente de Digg y pide que se clausure tu cuenta.",
-        "notes_pl": "Skontaktuj się z obsługą techniczną i poproś o usunięcie swojego konta",
+        "url": "http://digg.zendesk.com/hc/en-us/articles/233934647-How-do-I-delete-my-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://digg.zendesk.com/hc/en-us/articles/233934647-How-do-I-delete-my-account",
```

File: _data/sites.json [hunk 20]
```diff
@@ -1835,8 +1827,10 @@
 
     {
         "name": "Diigo",
-        "url": "https://www.diigo.com/",
+        "url": "http://diigo.com/setting",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://www.diigo.com/setting",
```

File: _data/sites.json [hunk 21]
```diff
@@ -2048,15 +2044,11 @@
...
-        "url": "http://www.dwolla.com/deactivate",
-        "difficulty": "hard",
-        "notes": "All accounts stay in their system for at least 3 years. Remove any banking information before you delete.",
-        "notes_fr": "Tous les comptes restent dans le système pour 3 ans au moins. Enlevez toutes les informations bancaires avant la suppression.",
-        "notes_de": "Alle Accounts bleiben 3 Tage lang in ihrem System. Lösche alle Bank-Informationen bevor du deinen Account löschst.",
-        "notes_it": "Tutti gli account rimangono nei loro sistemi per almeno 3 anni. Rimuovi qualsiasi credenziale bancaria prima di cancellarti.",
-        "notes_pt_br": "Todas as contas permanecem no sistema por pelo menos 3 anos. Remova qualquer informação de bancos antes de deletar.",
-        "notes_cat": "Tots els comptes romandran al sistema durant 3 anys. Esborri qualsevol informació bancaria abans de suprimir el compte.",
-        "notes_es": "Todas las cuentas permanecerán en el sistema durante 3 años. Borra toda información bancaria antes de eliminar tu cuenta.",
+        "url": "http://dwolla.com",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://www.dwolla.com/",
```

File: _data/sites.json [hunk 22]
```diff
@@ -2496,8 +2489,10 @@
 
     {
         "name": "Flattr",
-        "url": "https://flattr.com/",
+        "url": "http://flattr.com/settings/account/delete",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://flattr.com/settings/account/delete",
```

File: _data/sites.json [hunk 23]
```diff
@@ -2586,6 +2581,17 @@
         ]
     },
 
+    {
+        "name": "FreeBitco.In",
+        "url": "http://freebitco.in",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://freebitco.in",
```

File: _data/sites.json (hunk 12 above)
Comment (Nightfirecat) [reply _data/sites.json]: Also, it looks like the URL should be the following:
```suggestion
        "url": "https://suporte.neon.com.br/hc/pt-br/articles/360019117253-Quero-cancelar-minha-conta-Como-fa%C3%A7o",
```

File: _data/sites.json [hunk 24]
```diff
@@ -5236,6 +5249,17 @@
         ]
     },
 
+    {
+        "name": "Nubank",
+        "url": "http://blog.nubank.com.br/fechar-nuconta-e-facil",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://blog.nubank.com.br/fechar-nuconta-e-facil/",
```

File: _data/sites.json [hunk 25]
```diff
@@ -5812,6 +5839,19 @@
         ]
     },
 
+    {
+        "name": "Pluralsight",
+        "url": "http://help.pluralsight.com/help/how-do-i-delete-my-account",
```
Comment (Nightfirecat) [inline _data/sites.json]:
```suggestion
        "url": "https://help.pluralsight.com/help/how-do-i-delete-my-account",
```

**Comment by SnwMds** [comment]
Done, @Nightfirecat.

**Comment by tupaschoal** [comment]
Thanks a lot!

**Pull request merged**